.github/workflows/kitctl-test.yml

```diff
@@ -31,7 +31,7 @@ jobs:
           sudo mv kubectl /usr/local/bin/
       - name: Build kitctl, provision Kit Environment and KIT guest cluster.
         run: |
-          allVersions=("1-21" "1-22" "1-23" "1-24" "1-25")
+          allVersions=("1-21" "1-22" "1-23" "1-24" "1-25" "1-26" "1-27" "1-28" "1-29")
```

Comment:
Thanks for doing it.
We plan to deprecate the kitctl and those side of things very soon.
Let's not invest anymore time on these kitctl specific workflows.